content/mooreslaw-tutorial.ipynb

```diff
@@ -0,0 +1,807 @@
+{
```

Comment:
<p>It might be nice to add a quick link to the <code>plt.style</code>  documentation (the other commands are fairly standard so there's no need).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/numpy/numpy-tutorials/pull/31/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>